src/nidcpower/system_tests/test_system_nidcpower.py

```diff
@@ -243,3 +249,62 @@ def test_wait_for_event_with_timeout():
         with session.initiate():
             session.wait_for_event(nidcpower.Event.SOURCE_COMPLETE, 0.5)
 
+
+def test_commit(single_channel_session):
+    non_default_current_limit = 0.00021
+    single_channel_session.current_limit = non_default_current_limit
+    single_channel_session.commit()
```

Comment:
Don't need to commit, because the act of reading back an attribute is enough to guarantee that we wrote the attribute.